index.html

```diff
@@ -0,0 +1,856 @@
+            <h4>More in Space and Astronomy</h4>
+            <div class="articles">
+              <div class="article">
+                <figure>
+                  <img src="./assets/images/img-1.jpg" alt="article image" />
+                  <figcaption><small>NASA</small></figcaption>
+                </figure>
+                <h3>
+                  Astronauts Dock With Space Station After Historic SpaceX
+                  Launch
+                </h3>
+                <p>June 1</p>
+              </div>
+              <div class="article">
+                <figure>
+                  <img src="./assets/images/img-2.jpg" alt="article image" />
+                  <figcaption>
+                    <small>Rick Guidice/NASA Ames Research Center</small>
+                  </figcaption>
+                </figure>
+                <h3 class="article-title">
+                  Going Viral, or Not, in the Milky Way
+                </h3>
+                <p>June 2</p>
+              </div>
+              <div class="article">
+                <figure>
+                  <img src="./assets/images/img-3.jpg" alt="article image" />
+                  <figcaption>
+                    <small>John Raoux/Associated Press</small>
+                  </figcaption>
+                </figure>
+                <h3 class="article-title">
+                  SpaceX Lifts NASA Astronauts to Orbit, Launching New Era of
+                  Spaceflight
+                </h3>
+                <p>May 31</p>
+              </div>
+              <div class="article">
+                <figure>
+                  <img src="./assets/images/img-4.jpg" alt="article image" />
+                  <figcaption><small>Space X</small></figcaption>
+                </figure>
+                <h3 class="article-title">
+                  Now Boarding: SpaceX’s New Ride to Orbit for NASA Astronauts
+                </h3>
+                <p>May 30</p>
+              </div>
+              <div class="article">
+                <figure>
+                  <img src="./assets/images/img-5.jpg" alt="article image" />
+                  <figcaption>
+                    <small>Joe Raedle/Getty Images</small>
+                  </figcaption>
+                </figure>
+                <h3 class="article-title">
+                  How NASA’s Astronauts Became SpaceX’s Customers
+                </h3>
+                <p>May 28</p>
+              </div>
+              <div class="article">
+                <figure>
+                  <img src="./assets/images/img-6.jpg" alt="article image" />
+                  <figcaption><small>NASA</small></figcaption>
+                </figure>
+                <h3 class="article-title">
+                  NASA Names Dark Energy Telescope for Nancy Grace Roman
+                </h3>
+                <p>May 20</p>
+              </div>
+            </div>
+          </div>
+          <div class="rec-editor-pick">
```

Comment:
- Add some space below this `rec-editor-picks`. Right now it's sticking to the `footer`.  
  
  ![Annotation 2020-06-09 224045](https://user-images.githubusercontent.com/57726348/84178582-4955b900-aaa2-11ea-9960-590747bc25a1.jpg)